default.nix

```diff
@@ -1,9 +1,16 @@
 { nixpkgs ? <nixpkgs> }:
+let
+  nix-container-images = builtins.fetchTarball {
+    url = "https://github.com/nlewo/nix-container-images/archive/7577da87f7249442b51359e36a0f0493949a14d6.tar.gz";
```

Comment:
How do you intend to keep this up-to-date?

Reply:
For now it's manual, but an updater script would be nice

Reply:
What about using Niv?
I don't know if it is relevant  but we could also use Niv to fetch NixOS releases which seem to be fetched by the CI. Correct me if I'm wrong.

Reply:
yeah niv could work but the hard part is mostly around CI. make a cron-like job that updates and pushes updates if it works. Mic92 already did something like that for NUR so I might steal that.

Reply:
Also have this project for automated updates:

https://github.com/Mic92/iana-etc/blob/master/ci/deploy.sh

Setting up secrets is sometimes a hassle in travis, when you mix up travis.org and travis.com keys.